codec/reflectcodec/type_codec.go

```diff
@@ -29,6 +29,13 @@ var (
 
 var _ codec.Codec = (*genericCodec)(nil)
 
+func max(a, b int) int {
+	if a > b {
+		return a
+	}
+	return b
+}
```

Comment:
can we remove this and use `math.Max` from `utils/math/safe_math.go` instead?

Reply:
I wasn't familiar with that package. I just pushed a fix in 56c9ee14ad3974bf283f159e38ccd699a19baa5b